libs/agno/agno/utils/agent.py

```diff
@@ -370,3 +375,338 @@ def scrub_history_messages_from_run_output(run_response: Union[RunOutput, TeamRu
     # Remove messages with from_history=True
     if run_response.messages:
         run_response.messages = [msg for msg in run_response.messages if not msg.from_history]
+
+
+def get_run_output_util(
+    entity: Union["Agent", "Team"], run_id: str, session_id: Optional[str] = None
```

Comment:
`get_run_ouput`?

Reply:
Then it clashes with the name in the agent/team